docs/source/usage/workflows/boundaryConditions.rst

```diff
@@ -32,14 +32,13 @@ It is recommended to avoid, as possible, having particles in the field absorber
 Ideally, only particles leaving the simulation area are present there, on their way to be absorbed.
 Note that particle absorption happens at the external surface of the field absorber layer, matching the global simulation area border.
 
-The field absorber mechanism and user-controlled parameters depend on the field solver enabled.
+The field absorber mechanism and user-controlled parameters depend on the field absorber kind enabled.
 
-Field solvers ``Yee<>``, ``Lehe<>``, and ``ArbitraryOrderFDTD<>`` use the exponential damping absorber.
-Its parameters are controlled by :ref:`grid.param <usage-params-core>`.
+It is controlled by command-line option ``fieldAbsorber.kind``.
```

Comment:
maybe be `--fieldAbsorber.kind` even if you already mentioned that this is a command line parameter but in all other documentation we explicitly used `--`.

Could we add a sentence that the absorber is in all directions used were `--periodic` is set to zero.

Reply:
Adding `--`: okay, will do.
Regarding `--periodic` this is already written at start of this file. Do you think it needs further clarification here? I don't

Reply:
>  Do you think it needs further clarification here?

No, I missed the part at the top of the file. All is fine. +1